content/en/docs/concepts/storage/volumes.md

```diff
@@ -678,6 +678,10 @@ spec:
       type: FileOrCreate
 ```
 
+{{< note >}}
+Volumes' names must be different. For example: ```mydir``` & ```myfile``` as shown in the above [`FileOrCreate` configuration](#hostpath-fileorcreate-example)).
```

Comment:
I think it's not necessary to use ` ```code``` ` instead of `` `code` `` here.
```suggestion
Volumes' names must be different. For example: `mydir` & `myfile` as shown in the above [`FileOrCreate` configuration](#hostpath-fileorcreate-example)).
```

Reply:
@Sea-n, also, should we explicitly mention the same about volumeMounts ?
Thought: What about moving these changes to [Background ](https://kubernetes.io/docs/concepts/storage/volumes/#background)section instead at the beginning ?